sources/core/Stride.Core.Design/Stride.Core.Design.csproj

```diff
@@ -27,8 +27,9 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="2.3.2262-g94fae01e" />
-    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="2.1.0" />
-    <PackageReference Include="NuGet.Configuration" Version="5.8.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="6.0.0-preview.7.21377.19" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
This change is suspicious, ``Core.Design`` needs ``Newtonsoft.Json`` and it was previously packaged with the two other references ?

Reply:
yes, ```Newtonsoft.Json``` was bundled inside of ```"Microsoft.Extensions.DependencyModel" Version="2.1.0"```